inspire/dojson/hepnames/fields/bd1xx.py

```diff
@@ -304,14 +304,31 @@ def positions2marc(self, key, value):
 
 
 @hepnames.over('field_categories', '^65017')
-@utils.for_each_value
-@utils.filter_values
 def field_categories(self, key, value):
-    return {
-        'name': value.get('a').lower(),
-        'type': value.get('2'),
-    }
 
+    def get_value(value):
+        if isinstance(value, dict):
+            name = utils.force_list(value.get('a').lower())
```

Comment:
Better `value.get('a', '')`. Otherwise `.lower()` will crash when there is no `a`.